todoman/ui.py

```diff
@@ -343,21 +337,23 @@ def format_database(self, database):
 
 class PorcelainFormatter:
 
-    def compact(self, todo):
-        data = dict(
-            completed=todo.is_completed,
-            due=self.format_datetime(todo.due),
-            id=todo.id,
-            list=todo.list.name,
-            percent=todo.percent_complete,
-            summary=todo.summary,
-            # XXX: Move this into Todo itself and dedupe it
-            urgent=todo.priority not in [None, 0],
-        )
+    def compact_multiple(self, todos):
+        for todo in todos:
+            data = dict(
+                completed=todo.is_completed,
+                due=self.format_datetime(todo.due),
+                id=todo.id,
+                list=todo.list.name,
+                percent=todo.percent_complete,
+                summary=todo.summary,
+                # XXX: Move this into Todo itself and dedupe it
+                urgent=todo.priority not in [None, 0],
+            )
```

Comment:
The `compact` method is gone. Either we need to restore it or remove it from the other formatter.